content/blog/2019-06-12-taking-over-maint.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2019-06-12
+slug: taking-over-maint
+title: Taking over maintainence of a software package
```

Comment:
Add the outline / goals of the post to the end of the intro?

Reply:
what do you mean?